stdlib/logging/config.pyi

```diff
@@ -15,9 +16,47 @@ if sys.version_info >= (3, 7):
 else:
     _Path = StrPath
 
+_TKey = TypeVar("_TKey")
+_TValue = TypeVar("_TValue")
+
 DEFAULT_LOGGING_CONFIG_PORT: int
 RESET_ERROR: int  # undocumented
 IDENTIFIER: Pattern[str]  # undocumented
+_listener: Optional[Any]
+dictConfigClass = DictConfigurator
+
+class ConvertingMixin(Generic[_TKey, _TValue]):
+    def convert_with_key(self, key: _TKey, value: _TValue, replace: bool = ...) -> Dict[_TKey, _TValue]: ...
+    def convert(self, value: _TValue) -> Dict[_TKey, _TValue]: ...
+
+class ConvertingDict(ConvertingMixin[_TKey, _TValue]):
```

Comment:
`ConvertingDict` and `ConvertingList` are intended to generic, and I think it's good to be explicit about it:

```suggestion
class ConvertingDict(Generic[_TKey, _TValue], ConvertingMixin[_TKey, _TValue]):
```